bin/oneoff/import_unit_details.rb

```diff
@@ -191,6 +220,72 @@ def get_validated_lesson_pairs(script, chapters)
   validated_lesson_pairs
 end
 
+# Validates the following preconditions:
+
+# 1. Every CB chapter has a corresponding LessonGroup in Code Studio.
+# 2. If the CB unit uses chapters, then for each Code Studio chapter containing
+# lessons which have lesson plans, there is a corresponding CB chapter.
+# 3. For each LessonGroup with a corresponding CB chapter, the number of Lessons
+# in the Lesson Group which have lesson plans equals the number of lessons in
+# the CB chapter.
+# 4. we do not validate that the lesson names match like we did earlier, to
+# avoid duplicate warnings.
+#
+# Then, returns an array of pairs of LessonGroups and corresponding CB chapters.
+#
+# @param [Script] script - Code Studio Script/Unit object.
+# @param [Array<Hash>] - Array of CB chapters
+# @param [Array<Array.<Lesson, Hash>>] - Array of pairs of Code Studio Lesson
+#   objects and CB lessons.
+# @return [Array<Array<LessonGroup, Hash>>] lesson_pairs - Array of pairs of
```

Comment:
is this lesson_pairs or lesson_group_pairs?

Reply:
updated! thanks for catching.